db/mysql.js

```diff
@@ -471,6 +476,38 @@ module.exports = function (log, error) {
     return this.write(UPDATE_LOCALE, [data.locale, uid])
   }
 
+
+  // Update : accounts
+  // Set    : lockedAt = $2
+  // Where  : uid = $1
+  // Update : accountUnlockCodes
+  // Set    : unlockCode = $3
+  // Where  : uid = $1
+  var UPDATE_LOCKED_AT = 'CALL lockAccount_1(?, ?, ?)'
+
```

Comment:
Nit: This UPDATE_LOCKED_AT also inserts/replaces any `accountUnlockCodes`. Perhaps just LOCK_ACCOUNT would be more indicative.